Podcast/EpisodeTableViewCell.swift

```diff
@@ -0,0 +1,357 @@
+//
```

Comment:
Is this file different enough to warrant all the repetition with FeedTableViewCell? Could we combine the two files so that the cell could or could not have the social context bar?